### PR TITLE
Feature/Shutdown command

### PR DESCRIPTION
**Pull request by OverFloyd**
Added new in game command to close the client (I personally find it useful when testing new features and then fast closing via terminal interface) and reorganized Common interface a bit (renamed worldEntities/worldTileEntities to getWorldEntity etc to match getWorld getters).

### CODE REVIEW

**Comment by Finstaaa** [comment]
dab

File: src/main/java/dev/fiki/forgehax/main/Common.java [hunk 1]
```diff
@@ -85,33 +88,42 @@ static EventBus getEventBus() {
...
+    return !isInWorld() ? Stream.empty() : getWorld().getPlayers().stream();
+  }
+
+  static Stream<Entity> getWorldEntities() {
     return !isInWorld() ? Stream.empty()
         : StreamSupport.stream(getWorld().getAllEntities().spliterator(), false);
   }
 
-  static Stream<TileEntity> worldTileEntities() {
+  static Stream<TileEntity> getWorldTileEntities() {
```
Comment (fr1kin) [inline src/main/java/dev/fiki/forgehax/main/Common.java:120]: I know this is proper naming conventions but why refactor in this PR?

**Review by fr1kin — CHANGES_REQUESTED** [review]

File: src/main/java/dev/fiki/forgehax/main/Common.java (hunk 1 above)
Comment (OverFloyd) [reply src/main/java/dev/fiki/forgehax/main/Common.java:120]: it's my bad habit to always include stuff that don't belong to the main PR topic lol, if you don't want I can do another one for this specifically

Comment (OverFloyd) [reply src/main/java/dev/fiki/forgehax/main/Common.java:120]: also for this getter specifically and related I'd like to not return a stream but just the list itself as you could use it for more stuff then

Comment (fr1kin) [reply src/main/java/dev/fiki/forgehax/main/Common.java:120]: Yeah that's fine. Just create another function though

**Pull request closed without merging**